manual/en/setup.html

```diff
@@ -38,7 +38,7 @@ <h1>Setup</h1>
 WebGL cannot use images from your hard drive directly. That means
 in order to do development you need to use a web server. Fortunately
 development web servers are super easy to setup and use.</p>
-<p>First off if you'd like you can download this entire site from <a href="https://github.com/gfxfundamentals/threejsfundamentals/archive/gh-pages.zip">this link</a>.
+<p>First off if you'd like you can download this entire site from <a href="https://github.com/gfxfundamentals/threejsfundamentals/archive/48f9ae305b8767d0d54402d117395a20bcacffdf.zip">this link</a>.
```

Comment:
The thing is that the download item's name now links cryptic (the commit hash). Since the old link isn't broken, can't we just keep in?

Reply:
@Mugen87 I will close this since Ive got wrong SHA here, you would need a parent of [this commit](https://github.com/gfxfundamentals/threejsfundamentals/commit/60e277b9fe8830712f4ca7b31ef048bf4458e377)